pyiron_atomistics/lammps/base.py

```diff
@@ -464,7 +466,8 @@ def collect_h5md_file(self, file_name="dump.h5", cwd=None):
             ]
             indices = [indices_i.tolist() for indices_i in h5md["/particles/all/indices/value"]]
         with self.project_hdf5.open("output/generic") as h5_file:
-            h5_file["forces"] = np.array(forces)
+            # Store after converting to pyiron units
+            h5_file["forces"] = np.array(forces) / LAMMPS_UNIT_CONVERSIONS[self.input.control["units"]]["force"]
             h5_file["positions"] = np.array(positions)
```

Comment:
Good catch on converting forces! I believe we must also convert positions and cells with `LAMMPS_UNIT_CONVERSIONS[self.input.control["units"]]["distance"]`